google-cloud-datastore/pom.xml

```diff
@@ -38,12 +38,17 @@
       <version>1.3.0</version>
       <exclusions>
         <exclusion>
-          <artifactId>guava-jdk5</artifactId>
           <groupId>com.google.guava</groupId>
+          <artifactId>guava-jdk5</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-common-protos</artifactId>
+      <version>${generatedProto.version}</version>
+    </dependency>
```

Comment:
I think we should move this to an exclusion under datastore-v1-proto-client and add a dependency on proto-google-common-protos.